pkgs/applications/misc/gnuradio-ais/default.nix

```diff
@@ -26,11 +26,15 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  patches = [ ./fix-volk.patch ];
```

Comment:
Upstream hasn't had any activity in 2+yr. [Macports seems to have taken the same route and  included the patch with the build definition](https://github.com/macports/macports-ports/tree/master/science/gr-ais)